packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -166,15 +166,16 @@ export class FabricTestLedgerV1 implements ITestLedger {
     return `${this.envVars.get("FABRIC_VERSION")}`;
   }
 
-  public getDefaultMspId(): string {
-    return "Org1MSP";
+  public getDefaultMspId(organization: string): string {
```

Comment:
@AndreAugusto11 Adding a new and mandatory parameter to an existing method that is exported to the public API surface of the package is a breaking change that can only be done in major releases (e.g. when going from v1.x.x. to v2.0.0 or v3.0.0).

My recommendation is to revert `getDefaultMspId()` to how it was (and all the dozens of test cases that the compiler forced you to update as well) and add a brand new method called `capitalizedMspIdOfOrg(organizatoin: string)` which can then go and do the exact logic that you made `getDefaultMspId` do. Finally go ahead and use your new metohd where it's needed (which will probably be a few places only instead of having to update every single fabric test case)

Reply:
@petermetz the `getDefaultMspId()` method is being used by other methods (e.g. `enrollUser()` and `enrollAdmin()`), I would also need to change those to call the new method, but they depend on the new `organization` argument I added (as you identified below).
So in the app when I call `enrollUser()` or `enrollAdmin()` it will always use the default `getDefaultMspId()` unless I change it.

Reply:
@AndreAugusto11 

> @petermetz the getDefaultMspId() method is being used by other methods (e.g. enrollUser() and enrollAdmin()), 

I understand that it's not the least pleasant thing, but breaking changes are still a hard no unfortunately.

> I would also need to change those to call the new method, 

Doing that wouldn't work because we would be  back to the same conversation about those two methods the same way we are here now for `getDefaultMspId()`. :-)

What you can do instead is also introduce new versions of the `enrollX()` methods which allow the customizations. Much more work but it's worth it (in my humble opinion).

> but they depend on the new organization argument I added (as you identified below).

Yeah, funny how these things cascade together and from one tiny change you end up doing a relatively big refactor just to keep the ship straight. I sent a patch to demonstrate what I mean. Please apply it and then fix the remaining compiler errors (mostly just reverting the test cases to how they were)

Reply:
@petermetz I understand, definitely changing the interface of these methods is not the best thing to do.

I believe I got it now, thank you for the help with the patch!

Reply:
@AndreAugusto11 You are welcome! Thank you very much for this awesome contribution!